app/views/home/packages.html.erb

```diff
@@ -0,0 +1,59 @@
+  <!-- Packages -->
+  <section class="packages text-center bg-light">
+    <div class="container">
+      <h2 class="mb-5">Choose from one of our best packages</h2>
+      <div class="row">
+        <div class="col-lg-4">
+          <div class="mx-auto mb-5 mb-lg-0">
+            <div class="card text-center" style="width: 18rem;">
+              <div class="card-body">
```

Comment:
instead of writing multiple packages html, move common code to a partial and pass variables required